### PR TITLE
Add scroll to top button

### PR DESCRIPTION
**Pull request by laurakwhit**
## What was changed

This adds a button that appears on scroll for **all** pages to take the user back to the top of the page.

## Why?

It was requested.

## Checklist

1. Closes: Back to Top / Back to Bottom 

2. How was this tested:

- [ ] Verify button shows up on scroll down
- [ ] Verify button does not show up when user is at the top of the page
- [ ] Verify, when clicked, page is taken back to the top
- [ ] Verify button shows up on all pages that can scroll
- [ ] Verify layout for all pages looks as expected
3. Any docs updates needed? n/a

### CODE REVIEW

File: src/routes/__layout-root.svelte [hunk 1]
```diff
@@ -63,23 +63,25 @@
   import Banners from '$lib/components/banner/banners.svelte';
   import { ErrorBoundary } from '$lib/components/error-boundary';
   import PageTitle from '$lib/holocene/page-title.svelte';
+  import ScrollToTop from '$lib/holocene/scroll-to-top.svelte';
```
Comment (laurakwhit) [inline src/routes/__layout-root.svelte:66]: Wondering if we should add this and make updates to `__layout-import.svelte` as well 🤔

File: src/routes/__layout-root.svelte [hunk 2]
```diff
@@ -63,23 +63,25 @@
...
 
   export let user: User;
   export let uiVersionInfo: UiVersionInfo;
 </script>
 
 <PageTitle />
-<div class="flex h-screen w-screen flex-row">
+<div class="flex w-screen flex-row">
```
Comment (laurakwhit) [inline src/routes/__layout-root.svelte:73]: The `h-screen` was messing with the `on:scroll` here, hence it's removal.

Comment (Alex-Tideman) [reply src/routes/__layout-root.svelte:73]: I'm cool with this, but make sure to link with cloud ui and test it there to see if there's any wonkiness importing it in. That repo also uses several h-screen classes

Comment (laurakwhit) [reply src/routes/__layout-root.svelte:73]: Does `cloud-ui` use this layout? I've also made [updates](https://github.com/temporalio/cloud-ui/pull/247) directly to the layout in `cloud-ui`.

**Review by Alex-Tideman — APPROVED** [review]

**Pull request merged**